### ISSUE TITLE
Node Label Position Change (defaultLabelAlignment not working)

### ISSUE THREAD

**Issue by srinathganesh** [jacomyal/sigma.js#996]
I set `defaultLabelAlignment` and its not getting applied, anything I am doing wrong? refered it here https://github.com/jacomyal/sigma.js/pull/468/files

```
settings: {
            defaultLabelAlignment: 'left',
        }
```

full code

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Arrow Size</title>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/sigma.js/1.2.1/sigma.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/sigma.js/1.2.1/plugins/sigma.renderers.edgeLabels.min.js"></script>
</head>
<body>
    <div id="container">
        <style>
            #graph-container {
            top: 0;
            bottom: 0;
            left: 0;
            right: 0;
            position: absolute;
            }
        </style>
        <div id="graph-container"></div>
    </div>
</body>

<script>

    var i,
    s,
    N = 5,
    E = 6,
    g = {
      nodes: [],
      edges: []
    };

    // Generate a random graph:
    for (i = 0; i < N; i++)
      g.nodes.push({
        id: 'n' + i,
        label: 'Node ' + i,
        x: Math.random(),
        y: Math.random(),
        size: 3,
        color: '#666'
      });

    for (i = 0; i < E; i++)
      g.edges.push({
        id: 'e' + i,
        source: 'n' + (Math.random() * N | 0),
        target: 'n' + (Math.random() * N | 0),
        size: 1,
        color: '#ccc',
        type: 'arrow',
      });

    // Instantiate sigma:
    s = new sigma({
        graph: g,
        container: 'graph-container',
        renderer: {
            container: document.getElementById('graph-container'),
            type: 'canvas',
        },
        settings: {
            minNodeSize: 8,
            maxNodeSize: 20,
            maxEdgeSize: 10,
            doubleClickEnabled: false,
            defaultLabelAlignment: 'left',
        }
    });
</script>
</html>
```
